boule/sphere.py

```diff
@@ -0,0 +1,105 @@
+"""
+Module for defining and setting the reference sphere.
+"""
+import attr
+import numpy as np
+
+
+# Don't let ellipsoid parameters be changed to avoid messing up calculations
+# accidentally.
+@attr.s(frozen=True)
+class Sphere:
+    """
+    Reference sphere.
+
+    Implements the representation of ellipsoids with zero flattenning as spheres.
```

Comment:
```suggestion
    Implements the representation of ellipsoids with zero flattening as spheres.
    This is required since some computations can fail if the flattening is too small.
```